plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -201,6 +201,16 @@ class CreateServiceModalForm extends Component {
       return true;
     }
 
+    if (nextProps.expandAdvancedSettings) {
+      nextProps.resetExpandAdvancedSettings();
+    }
+
+    if (
+      nextProps.expandAdvancedSettings !== this.props.expandAdvancedSettings
+    ) {
+      return true;
```

Comment:
can we get rid of `shouldComponentUpdate` here?
i don't see this being a performance issue, thus being a good spot to reduce complexity.

Reply:
There might be a way to remove it, but it was not added in this PR, so I would say removing it is out of scope here.